client/me/two-step/index.jsx

```diff
@@ -89,9 +90,21 @@ class TwoStep extends Component {
 		return <Security2faBackupCodes />;
 	};
 
+	renderEnhancedSecuritySetting = () => {
+		if (
+			! isEnabled( 'two-factor/enhanced-security' ) ||
+			this.props.isFetchingUserSettings ||
```

Comment:
Do we mean to render the component if `isFetchingUserSettings` is true though? I see we're disabling the toggle field when fetching user settings, so we should be good to render it in that case?

Reply:
I was just following the other `render*` callbacks which don't render when `isFetchingUserSettings` is `true`. Is there a preferred option between not rendering vs disabled the setting?

Reply:
That's totally fine to me, but then we don't need to disable the toggle in that case because a disabled toggle will never render anyway.

Reply:
Sounds good, removed it in 02c166fc41